### PR TITLE
Naming convention

### PR DESCRIPTION
**Pull request by kosecki123**
This PR contains various changes for variable names etc. The purpose is to keep ubiquitous in debt smart contract

### CODE REVIEW

File: contracts/DebtToken.sol [hunk 1]
```diff
@@ -155,93 +173,86 @@ contract DebtToken is Ownable {
...
-  */
-
   /**
   calculate the total number of passed interest cycles and coin value
   */
   function calculateInterestDue() public constant returns(uint256 _coins,uint256 _cycle){
-    if(!loanMature())
-      return (0,0);
-    else if(balances[debtOwner] == 0)
+    if(!isTermOver() || !isLoanFunded())
```
Comment (adibas03) [inline contracts/DebtToken.sol]: `!isLoanFunded()` use on this line is redundant, as `!isTermOver()` has implicit check for isFunded

Comment (kosecki123) [reply contracts/DebtToken.sol]: Agree

File: contracts/DebtToken.sol [hunk 2]
```diff
@@ -155,93 +173,86 @@ contract DebtToken is Ownable {
...
+    Transfer(lender,owner,totalSupply);//Allow funding be tracked
   }
 
   /**
   Fallback function
   */
   function() public payable{
     require(initialSupply > 0);//Stop the whole process if initialSupply not set
-    if(msg.sender == owner && balances[msg.sender] == 0)
+    if(isBorrower())
```
Comment (adibas03) [inline contracts/DebtToken.sol]: we need to check for the fund state ` &&  isLoanFunded()`

File: contracts/DebtToken.sol [hunk 3]
```diff
@@ -155,93 +173,86 @@ contract DebtToken is Ownable {
...
       refundLoan();
-    else if(isDebtOwner(msg.sender) && balances[msg.sender] == 0)
+    else if(isLender())
```
Comment (adibas03) [inline contracts/DebtToken.sol]: fund state check needed ` && !isLoanFunded()`

Comment (kosecki123) [reply contracts/DebtToken.sol]: Both `fundLoan` and `refundLoan` has `require` statement for isLoanFunded so it was redundant here.

Comment (adibas03) [reply contracts/DebtToken.sol]: That is right,
missed that

**Pull request merged**